src/test/kotlin/uk/gov/dluhc/printapi/rest/GenerateTemporaryCertificateExplainerDocumentIntegrationTest.kt

```diff
@@ -148,8 +147,5 @@ internal class GenerateTemporaryCertificateExplainerDocumentIntegrationTest : In
 
         val pdfContent = result.responseBody
         assertThat(pdfContent).isNotNull
-        PdfReader(pdfContent).use { reader ->
```

Comment:
The resultant PDF that is generated and returned from the REST API no longer has form fields